packages/datatrak-web/src/features/Survey/SurveyScreen.tsx

```diff
@@ -144,77 +153,79 @@ export const SurveyScreen = () => {
   return (
     <>
       <SurveyToolbar />
-      <Container>
-        <SideMenu />
-        <Paper>
-          <FormProvider {...formContext}>
-            <StyledForm onSubmit={onStepForward} noValidate>
-              <FormScrollBody>
-                <ScreenHeading variant="h2">{screenHeader}</ScreenHeading>
-                {displayQuestions.map(
-                  ({
-                    questionId,
-                    questionCode,
-                    questionText,
-                    questionType,
-                    questionOptions,
-                    config,
-                    questionLabel,
-                    validationCriteria,
-                    detailLabel,
-                    questionOptionSetId,
-                    questionNumber,
-                  }) => {
-                    if (validationCriteria?.mandatory === true) {
-                      console.log('mandatory question', questionCode);
-                    }
-                    return (
-                      <QuestionWrapper
-                        key={questionId}
-                        $isInstruction={questionType === 'Instruction'}
-                      >
-                        {questionNumber && (
-                          <QuestionNumber id={`question_number_${questionId}`}>
-                            {questionNumber}
-                          </QuestionNumber>
-                        )}
-                        <SurveyQuestion
-                          detailLabel={detailLabel}
-                          id={questionId}
-                          code={questionCode}
-                          name={questionCode}
-                          type={questionType}
-                          text={detailLabel || questionText}
-                          options={questionOptions}
-                          config={config}
-                          label={questionLabel || questionText}
-                          optionSetId={questionOptionSetId}
-                        />
-                      </QuestionWrapper>
-                    );
-                  },
-                )}
-              </FormScrollBody>
-              <FormActions>
-                <Button
-                  onClick={onStepPrevious}
-                  startIcon={<ArrowBackIosIcon />}
-                  variant="text"
-                  color="default"
-                >
-                  Back
-                </Button>
-                <ButtonGroup>
-                  <Button variant="outlined" to={ROUTES.SURVEY_SELECT}>
-                    Cancel
+      <Wrapper>
+        <SurveySideMenu />
+        <Container $sideMenuOpen={sideMenuOpen}>
+          <Paper>
+            <FormProvider {...formContext}>
```

Comment:
I think this is my bad from when I set this up but the FormProvider needs to go outside of the SurveySideMenu so that the Side menu can have access to form state